packages/expect/src/matchers.js

```diff
@@ -87,7 +87,15 @@ const matchers: MatchersObject = {
   toBeCloseTo(actual: number, expected: number, precision?: number = 2) {
     const secondArgument = arguments.length === 3 ? 'precision' : null;
     ensureNumbers(actual, expected, '.toBeCloseTo');
-    const pass = Math.abs(expected - actual) < Math.pow(10, -precision) / 2;
+
+    let pass = false;
+    //If both actual and expected are Infinity it is true they are close
```

Comment:
Nit - mathematically speaking it's not true that two infinities are close to each other, the comment should read something like "Check that both are not equal to the Infinity global"

Reply:
In the changelog, the items in the list that are **`[BREAKING]`**  are listed first. Easiest fix would be to move your change to the bottom of the *Features* section